rollup.config.js

```diff
@@ -15,9 +15,11 @@ const commitHash = (function () {
 	}
 })();
 
+const now = new Date(process.env.SOURCE_DATE_EPOCH ? (process.env.SOURCE_DATE_EPOCH * 1000) : new Date().getTime()).getTime();
```

Comment:
The previous version would put a human readable date into the banner. Would it be ok with your efforts to remove the final `getTime()` here? Or maybe use `toISOString` instead to get a more standardised output?

Reply:
I'm easy! :) (indeed, see the linked Debian bug for a patch for the previous version)

Reply:
Ah, I see! I guess I would go with the original suggestion then.